provider/source/data/debug/datetime/patterns/date/chinese/v1/ej/en-001.json

```diff
@@ -11,9 +11,9 @@
   "elements": [
     "cccc, h a",
     "ccc, h a",
-    "EEEE h:m a",
-    "E h:mm a",
-    "EEEE h:m:s a",
-    "E h:mm:ss a"
+    "cccc h:m a",
+    "ccc h:mm a",
+    "cccc h:m:s a",
+    "ccc h:mm:ss a"
```

Comment:
Most of these changes are wrong. `cccc, h a` is the "generated" pattern, and `EEEE h:m a` is the "real" pattern directly from availableFormats. Hmm.